deploy/template.yaml

```
@@ -1065,6 +1065,50 @@ Resources:
         - AddProvisionedConcurrency
         - ProvisionedConcurrentExecutions: !FindInMap [ EnvironmentConfiguration, !Ref AWS::AccountId, provisionedConcurrency ]
         - !Ref AWS::NoValue
+        -
+  SelectCriFunction:
+    Type: AWS::Serverless::Function
+    DependsOn:
+      - SelectCriFunctionLogGroup
+    Properties:
+      # checkov:skip=CKV_AWS_115: We do not have enough data to allocate the concurrent execution allowance per function.
+      # checkov:skip=CKV_AWS_116: Lambdas invoked via API Gateway do not support Dead Letter Queues.
+      # checkov:skip=CKV_AWS_117: Lambdas will migrate to our own VPC in future work.
+      FunctionName: !Sub "evaluate-gpg45-scores-${Environment}"
+      Handler: uk.gov.di.ipv.core.selectcri::handleRequest
```

Missing class name here
```suggestion
      Handler: uk.gov.di.ipv.core.selectcri.SelectCriHandler::handleRequest
```